backend/src/seed/factories/index.js

```diff
@@ -73,6 +73,7 @@ export default function Factory(options = {}) {
       const { factory, mutation, variables } = this.factories[node](args)
       if (factory) {
         this.lastResponse = await factory({ args, neodeInstance })
+        return this.lastResponse
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-11T14:45:57Z" title="Thursday, July 11th 2019, 4:45:57 pm +02:00">Jul 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -73,6 +73,7 @@ export default function Factory(options = {}) {
       const { factory, mutation, variables } = this.factories[node](args)
       if (factory) {
         this.lastResponse = await factory({ args, neodeInstance })
+        return this.lastResponse
```


what's the point of saving this to `this.lastResponse`
is this something to do with `neode`?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-11T19:56:42Z" title="Thursday, July 11th 2019, 9:56:42 pm +02:00">Jul 11, 2019</time>_

---


Ah, this is the messy part of the refactoring. The factories are currently in a dangerous in-between state. Some of them return a `graphql` mutation while the recently updated factories return a function.

I'm returning `this.lastResponse` here, because it's a neode type and we can call `relateTo` etc. on it. We cannot call this method on a plain JS object.